source/collectBy.js

```diff
@@ -33,14 +33,6 @@ import _reduce from './internal/_reduce.js';
  *      //   [ {type: 'dinner', item: '🍝'} ] ]
  */
 var collectBy = _curry2(function collectBy(fn, list) {
-  var group = _reduce(function(o, x) {
-    var tag = fn(x);
-    if (o[tag] === undefined) { o[tag] = []; }
-    o[tag].push(x);
-    return o;
-  }, {}, list);
-  var newList = [];
-  for (var tag in group) { newList.push(group[tag]); }
-  return newList;
+  return Object.values(groupBy(fn, list));
```

Comment:
@CrossEye: IE does not have `Object.values`. Do we care about IE compatibility at this point?

Reply:
Let's stick with our ES3 compatibility through 1.0, then go wild with modern ES6 stuff post-1.0

Reply:
@ycmjason I think if you change `Object.values` with `R.values` we should be able to reconsider this.

Reply:
@customcommander done! 😄